test/AsmJs/rlexe.xml

```diff
@@ -522,6 +524,8 @@
     <default>
       <files>MathBuiltinsCall.js</files>
       <baseline>MathBuiltinsCall.baseline</baseline>
+      <!-- mac gives has some test which gives different precision result, so exclude it -->
```

Comment:
nit: open an issue to track these?

Reply:
I can, but do you think it's worth tracking? this is difference in precision with result of atan, which spec says is implementation defined approximation.

Reply:
I see. I thought only some tests have difference but others don't. If this is not the case, well, you are right, no point tracking it!

Generally speaking;
What we did with some of the date time and number formatting testing was to enable-disable per platform. i.e. `CH` supports the line below that checks if the system running the test is a macOS.

```
var isMac = (WScript.Platform && WScript.Platform.OS == 'darwin');
```